auditlog_security/models/__init__.py

```diff
@@ -0,0 +1,6 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from . import auditlog_rule
+from . import auditlog_line_access_rule
```

Comment:
Upon installing the module I saw: `Field ir.rule.auditlog_line_access_rule_id with unknown comodel_name 'auditlog.access.line.rule'` - maybe that goes away when the order here should be switched?

Reply:
not sure, but makes sense. switched.

Reply:
Turned out to be a typo, see my other comment.